### ISSUE TITLE
Absence of index page

### ISSUE THREAD

**Issue by kambata940** [relax/relax#399]
IMO the absence of index page is confusing. At least for me, in development mode took me some time to realize that I should open `/admin`. I guess the reason behind that is because you can build page through the CMS and publish it as Index page.
I offer in development the index to redirect to `/admin` in absence of published Index page.

**Comment by bruno12mota** [comment]
You're right. I do suggest redirecting to `/admin/init` when there is no users. When there's at least one user index should return 404 if no page is assigned as homepage.

**Comment by kambata940** [comment]
Why we care if there are more than one users? IMO we shoud check if the request is authenticated and if there is page assigned as homepage.

**Comment by bruno12mota** [comment]
That's fine as well. So:

* if authenticated - redirect to `/admin`
* if not authenticated
  * if no users in the system - redirect to `/admin/init`
  * if system has users - 404 page

What do you think?
